### PR TITLE
updated_chart_v4

### PR DESCRIPTION
**Pull request by nshankar13**
Updated the osm chart to version 4. Will run pipeline to deploy this to staging ACR

### CODE REVIEW

File: charts/osm/values.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ OpenServiceMesh:
   image:
     registry: openservicemesh
     pullPolicy: IfNotPresent
-    tag: v0.3.0
+    tag: v0.4.0
```
Comment (ritazh) [inline charts/osm/values.yaml:9]: Can you pls confirm the image tag and the chart version and appVersion are the only updates in the osm chart going from https://github.com/openservicemesh/osm/releases/tag/v0.3.0 to https://github.com/openservicemesh/osm/releases/tag/v0.4.0

Comment (nshankar13) [reply charts/osm/values.yaml:9]: The previous version of the chart that I had was from main, not the v0.3.0 release. So these were the only differences from the osm chart in main.

Comment (ritazh) [reply charts/osm/values.yaml:9]: Got it. Let's always use a released version of osm from now on.

**Review by ritazh — APPROVED** [review]
lgtm

**Pull request merged**